apps/ui-storybook/stories/sidebar.stories.ts

```diff
@@ -242,13 +245,12 @@ type Story = StoryObj<{}>;
 				<!-- Rail for resizing -->
 				<button hlmSidebarRail></button>
 			</hlm-sidebar>
-
-			<!-- <main hlmSidebarInset></main> -->
-			<!-- Main content -->
-			<div class="flex h-screen w-full flex-col items-center justify-center">
-				<button hlmSidebarTrigger><span class="sr-only"></span></button>
-				<h1 class="text-2xl font-bold">Hello World</h1>
-			</div>
+			<main hlmSidebarInset>
```

Comment:
Using hlmSidebarWrapper and hlmSidebarInset in the story.
hlmSidebarWrapper is part of the `SidebarProvider` in shadcn/ui